tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_door.py

```diff
@@ -0,0 +1,22 @@
+from metaworld.envs.mujoco.env_dict import ALL_ENVIRONMENTS
+from metaworld.policies.mujoco.sawyer_xyz.sawyer_door import SawyerDoorOpenV1Policy
+from metaworld.policies.mujoco.sawyer_xyz.sawyer_door_close import SawyerDoorCloseV1Policy
+from tests.metaworld.envs.mujoco.sawyer_xyz.base import scripted_policy_test
+
+
+def test_scripted_open():
```

Comment:
in the future, you can use a parameterized test to run all of these automagically.